src/database/sqlite/mod.rs

```diff
@@ -1089,4 +1121,157 @@ mod tests {
 
         fs::remove_dir_all(&tmp_dir).unwrap();
     }
+
+    #[test]
+    fn sqlite_list_txids() {
+        let (tmp_dir, _, _, db) = dummy_db();
+
+        {
+            let mut conn = db.connection().unwrap();
+
+            let coins = [
+                Coin {
+                    outpoint: bitcoin::OutPoint::from_str(
+                        "6f0dc85a369b44458eba3a1f0ea5b5935d563afb6994f70f5b0094e05be1676c:1",
+                    )
+                    .unwrap(),
+                    block_height: None,
+                    block_time: None,
+                    amount: bitcoin::Amount::from_sat(98765),
+                    derivation_index: bip32::ChildNumber::from_normal_idx(10).unwrap(),
+                    is_change: false,
+                    spend_txid: None,
+                    spend_block: None,
+                },
+                Coin {
+                    outpoint: bitcoin::OutPoint::from_str(
+                        "c449539458c60bee6c0d8905ba1dadb20b9187b82045d306a408b894cea492b0:2",
+                    )
+                    .unwrap(),
+                    block_height: Some(101_095),
+                    block_time: Some(1_121_000),
+                    amount: bitcoin::Amount::from_sat(98765),
+                    derivation_index: bip32::ChildNumber::from_normal_idx(100).unwrap(),
+                    is_change: false,
+                    spend_txid: None,
+                    spend_block: None,
+                },
+                Coin {
+                    outpoint: bitcoin::OutPoint::from_str(
+                        "f0801fd9ca8bca0624c230ab422b2e2c4c8dc995e4e1dbc6412510959cce1e4f:3",
+                    )
+                    .unwrap(),
+                    block_height: Some(101_099),
+                    block_time: Some(1_122_000),
+                    amount: bitcoin::Amount::from_sat(98765),
+                    derivation_index: bip32::ChildNumber::from_normal_idx(1000).unwrap(),
+                    is_change: false,
+                    spend_txid: Some(
+                        bitcoin::Txid::from_str(
+                            "0c62a990d20d54429e70859292e82374ba6b1b951a3ab60f26bb65fee5724ff7",
```

Comment:
nit: would have been nice to have another coin have this as its deposit txid. But i'll do that in the functional test anyways.